packages/hoverable/src/with-hover.spec.tsx

```diff
@@ -98,4 +99,79 @@ describe('withHover HOC', () => {
             expect(nestedElement).not.toHaveStyle(nestedHoverStyle);
         });
     });
+    describe('Children props', () => {
+        const nestedTestColor = 'black';
+
+        it('Should nested component have isActive prop from the root element, if children has hoverColor Prop', () => {
```

Comment:
A description message should be like it is started with it

so `it should nested component have - is not grammarly correct.`

it relates to all of the `it` sections below